relay-general/src/pii/convert.rs

```diff
@@ -707,31 +707,33 @@ THd+9FBxiHLGXNKhG/FRSyREXEt+NyYIf/0cyByc9tNksat794ddUqnLOg0vwSkv
         assert_annotated_snapshot!(data);
     }
 
+    macro_rules! sanitize_credit_card_within_value_test {
```

Comment:
`insta` has changed the way it infers snapshot names. Before, it used the thread name from `cargo test`, which was set to the function. Since that was too brittle, it now uses the name of the function where `insta::assert_*` is called. 

To keep the snapshot file names the same, we now use a macro rather than a subfunction.